### PR TITLE
OSDOCS-484: Include Azure persistent storage.

### PR DESCRIPTION
**Pull request by huffmanca**
Included draft of Azure disk persistent storage.

This is for OS 4.2 only.

### CODE REVIEW

**Comment by huffmanca** [comment]
@openshift/team-documentation Can someone review this one?

File: storage/persistent-storage/persistent-storage-azure.adoc [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Azure Disk volumes can be provisioned dynamically.
+Persistent volumes are not bound to a single project or namespace; they can be
+shared across the {product-title} cluster.
+Persistent volume claims are specific to a project or namespace and can be
+requested by users.
+
+[IMPORTANT]
+====
+High-availability of storage in the infrastructure is left to the underlying
```
Comment (bergerhoffer) [inline storage/persistent-storage/persistent-storage-azure.adoc]: High-availability -> High availability

File: modules/storage-azure-create-storage-class.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage-azure.adoc
+
+
+
+[id="storage-create-azure-storage-class_{context}"]
+= Creating the Azure storage class
+
+StorageClasses are used to differentiate and delineate storage levels and
```
Comment (bergerhoffer) [inline modules/storage-azure-create-storage-class.adoc]: Is this correct as "StorageClasses"? Or should it just be "Storage classes" like elsewhere?

File: modules/storage-azure-create-storage-class.adoc [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+.. Enter a name to reference the storage class.
+
+.. Enter an optional description.
+
+.. Select the reclaim policy.
+
+.. Select `kubernetes.io/azure-disk` from the drop down list.
+
+... Enter the storage account type. This corresponds to your Azure
+storage account Sku tier. Valid options are `Premium_LRS`, `Standard_LRS`,
```
Comment (bergerhoffer) [inline modules/storage-azure-create-storage-class.adoc]: Pretty sure Sku should be SKU

File: modules/storage-persistent-storage-azure-volume-format.adoc [hunk 4]
```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage-azure.adoc
+
+[id="volume-format-azure_{context}"]
+= Volume Format
```
Comment (bergerhoffer) [inline modules/storage-persistent-storage-azure-volume-format.adoc]: Should this be sentence case, or is "Volume Format" a thing?

File: modules/storage-persistent-storage-pv.adoc [hunk 5]
```diff
@@ -45,7 +45,7 @@ once it is released.
 - HostPath
 - iSCSI
 - NFS
-// - Azure Disk
+- Azure Disk
```
Comment (bergerhoffer) [inline modules/storage-persistent-storage-pv.adoc]: It looked like this list was previously alphabetical. Should Azure Disk go below AWS?

File: storage/persistent-storage/persistent-storage-azure.adoc [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+storage provider.
+====
+
+.Additional References
+
+link:https://azure.microsoft.com/en-us/services/storage/disks[Microsoft Azure Disk]
```
Comment (bergerhoffer) [inline storage/persistent-storage/persistent-storage-azure.adoc]: This should be bulleted.

File: storage/persistent-storage/persistent-storage-azure.adoc [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+Persistent volume claims are specific to a project or namespace and can be
+requested by users.
+
+[IMPORTANT]
+====
+High-availability of storage in the infrastructure is left to the underlying
+storage provider.
+====
+
+.Additional References
```
Comment (bergerhoffer) [inline storage/persistent-storage/persistent-storage-azure.adoc]: Sentence case.

File: storage/persistent-storage/persistent-storage-azure.adoc (hunk 1 above)
Comment (huffmanca) [reply storage/persistent-storage/persistent-storage-azure.adoc]: Corrected.

File: modules/storage-azure-create-storage-class.adoc (hunk 2 above)
Comment (huffmanca) [reply modules/storage-azure-create-storage-class.adoc]: Checking the glossary it looks like `StorageClasses` is correct.

File: modules/storage-azure-create-storage-class.adoc (hunk 3 above)
Comment (huffmanca) [reply modules/storage-azure-create-storage-class.adoc]: Corrected.

File: modules/storage-persistent-storage-azure-volume-format.adoc (hunk 4 above)
Comment (huffmanca) [reply modules/storage-persistent-storage-azure-volume-format.adoc]: Adjusted to sentence case.

File: modules/storage-persistent-storage-pv.adoc (hunk 5 above)
Comment (huffmanca) [reply modules/storage-persistent-storage-pv.adoc]: You're right - I made this alphabetical in 4.1 from 3.11, and moved this line.

File: storage/persistent-storage/persistent-storage-azure.adoc (hunk 6 above)
Comment (huffmanca) [reply storage/persistent-storage/persistent-storage-azure.adoc]: Added a bullet.

File: storage/persistent-storage/persistent-storage-azure.adoc (hunk 7 above)
Comment (huffmanca) [reply storage/persistent-storage/persistent-storage-azure.adoc]: Corrected.

**Comment by vikram-redhat** [comment]
@liangxia please review.

**Comment by liangxia** [comment]
LGTM (from QE)

**Pull request merged**